[Acceptance Guide] refactored to promote partner acceptance more prominently. Also Testing section added. (#116)
UnumID/UnumID.github.io: docs/acceptance-guide.mdx

```text title="Sandbox User Wallet Authentication OTP"
111111
```
:::

### Partner Acceptance
Partner Issuer Acceptance is enabled by a partnered issuer brand that will integrate more closely with you - ensuring the users will have the desired credentials for your brand's onboarding. This is generally the most popular integration type because of the parallels to traditional affiliate partnerships. However, this case supercharges the traditional flow by allowing users to accept promotions with 1-click.

### General Acceptance

> General Issuer Acceptance references the integration type where users may not be coming from a particular issuing brand. This is the flexible as it does not require any pre-arranged partnerships. However, it does require you to check whether the user has the necessary credentials for your brand's onboarding.

Author: Nice. Reads much better.

  <p>
    This demo shows Kredita accepting Hooli credentials in order to gain 1-Click
    IDV capabilities when it's available. They collect email and phone from the
    user (Richard) and check if the user has matching credentials. The user
    does, so Kredita lets them verify with 1-Click.
  </p>
  <iframe
    src='https://marvelapp.com/prototype/9cab6ai?emb=1&iosapp=false&frameless=false'
    width='453'
    height='864'
    allowTransparency='true'
    frameborder='0'
  ></iframe>
</details>

## Implementation

In all cases, you will need to configure a web client page to receive the `sharedCredentialsUuid` query param via a redirect from the Verified Inc. Wallet. Once the query param is retrieved it should be sent to your backend where it is used to call [`/sharedCredentials/{sharedCredentialsUuid}`](/api-overview#get-shared-credentials) to get the credentials to onboard the user. 

### Partner Acceptance

In this case, as the accepting <Tip type="brand">brand</Tip>, you only need to handle the two steps ([1](acceptance-guide#3-capture-query-param-value)) and ([2](/acceptance-guide#4-retrieve-shared-credentials-data)) below.

> Another critical component of this flow is that the issuer specifies your brand `uuid` as the partner when issuing credentials to the user. This effectively ensures the user will have the necessary credentials, and it allows you to skip steps ([0.a](/acceptance-guide#0a-check-user-credentials)) and ([0.b](/acceptance-guide#0b-redirect-user-to-verified-inc-wallet)). Because the User starts in the issuing brand's UX in this type of flow, the issuer handles the redirect in step ([0.b](/acceptance-guide#0b-redirect-user-to-verified-inc-wallet)). This is why you must pre-define your desired `redirectUrl` with us. For more information, see the [partner issuance](/issuance-guide#partner-issuance) section of the issuance guide.

Author: Yeah I suppose you are right. I think this detail does need to be mentioned in the Integration Testing section though.

It's easiest to extract the data from each credential using the corresponding [schema](/schema). The extracted data is verified according to the issuer of the credential, but you can perform additional verification steps if you like.

<CredentialUsageExample/>

✅ **_You're done!_**

### General Acceptance

> Because the issuer is not issuing credentials with your brand in mind and handling the redirect to the Verified Inc. Wallet, **you need to check whether the user has matching <Tip type="credential">credentials</Tip>.** If they do, they can share all of those credentials with 1-click.

Reviewer: ```suggestion
Because the issuer is not issuing credentials with your brand in mind, **you need to check whether the user has matching <Tip type="credential">credentials</Tip>.** If they do, they can share all of those credentials with 1-click. **You will also need to direct the user to the Verified Inc. Wallet.**
```
